### PR TITLE
Fixes #17124 - install capsule parser cache

### PR DESCRIPTION
**Pull request by beav**
There were some fixes in master for capsule parser cache, this commit
is a smaller version to just pull what's needed to build the capsule
parser cache for puppet 4.

### CODE REVIEW

**Comment by beav** [comment]
this depends on https://github.com/Katello/katello-installer/pull/426

**Comment by stbenjam** [comment]
#426 merged, restarted tests

File: config/capsule.yaml [hunk 1]
```diff
@@ -8,13 +8,133 @@
...
   :colors: true
   :order:
     - certs
     - capsule
+  :mapping:
+    :foreman::cli:
+      :dir_name: foreman
+      :manifest_name: cli
+      :params_name: cli/params
+    :foreman::plugin::bootdisk:
```
Comment (stbenjam) [inline config/capsule.yaml]: Hm, these shouldn't be here in the capsule scenario, these are all foreman plugins. What added them?

Comment (stbenjam) [inline config/capsule.yaml]: I think only the foreman_proxy plugins should be here

**Comment by beav** [comment]
updated to remove extraneous declarations in `capsule.yaml`

**Comment by stbenjam** [comment]
ACK

**Comment by beav** [comment]
@stbenjam thanks!

**Pull request merged**